### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by SaraCah**
Fixed the relationship "Is controlled by" to just say "Controlled by" and the other relationships. Changed the order of the mandate and function relationships. Also fixed the view digital copy but probably needs some better eyes.

### CODE REVIEW

File: config/config.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # create database qsa_public character set UTF8mb4 collate utf8mb4_bin;
 # grant all on qsa_public.* to 'qsa'@'localhost' identified by 'qsa123';
 
-# AppConfig[:db_url] = "jdbc:mysql://localhost:3306/qsa_public?useUnicode=true&characterEncoding=UTF-8&user=qsa&password=qsa123&serverTimezone=UTC"
+AppConfig[:db_url] = "jdbc:mysql://localhost:3306/qsa_public?useUnicode=true&characterEncoding=UTF-8&user=sara&password=hannah&serverTimezone=UTC"
```
Comment (jambun) [inline config/config.rb]: No changes to config.rb please

Comment (SaraCah) [reply config/config.rb]: Fixed up! Sorry always keep forgetting

File: qsa-public-spa/src/utils/typeResolver.ts [hunk 2]
```diff
@@ -70,7 +70,8 @@ export const snakeToUppercaseInitials = (s: string): string => {
   return s
     .split('_')
     .map(uppercaseInitials)
-    .join(' ');
+    .join(' ')
+    .replace("Is", "");
```
Comment (jambun) [inline qsa-public-spa/src/utils/typeResolver.ts]: What's the goal here? Do we want to change `Is controlled by` to `Controlled by` everywhere? or just in the public app?

If everywhere, then we should make the change in the series_system plugin - I think it's just a locale change for the relator.

Comment (SaraCah) [reply qsa-public-spa/src/utils/typeResolver.ts]: The change is only wanted for the public site.

**Comment by jambun** [comment]
Looks like there's a conflict @SaraCah. Do you mind taking a look?

**Comment by SaraCah** [comment]
Fixed the conflicts by accepting incoming changes on item

**Comment by jambun** [comment]
Many thanks @SaraCah

**Pull request merged**